imagetest/fixtures.go

```diff
@@ -156,3 +159,30 @@ func (t *TestVM) EnableSecureBoot() {
 		}
 	}
 }
+
+// AddTestUser generate ssh key pair and return public key.
+func (t *TestVM) AddTestUser(user string) ([]byte, error) {
+	email := fmt.Sprintf("%s@example.com", user)
+	keyFileName := t.testWorkflow.setKeyFileName()
+	if keyFileName == "" {
+		return nil, fmt.Errorf("no key file found for user")
+	}
+	commandArgs := []string{"-t", "rsa", "-C", email, "-f", keyFileName, "-N", "", "-q"}
+	cmd := exec.Command("ssh-keygen", commandArgs...)
+	if err := cmd.Run(); err != nil {
+		return nil, err
+	}
+	publicKey, err := ioutil.ReadFile(keyFileName + ".pub")
+	if err != nil {
+		return nil, err
+	}
+	return publicKey, nil
+}
+
+func randNum() int {
+	min := 0
+	max := 100
+	delta := max - min
+
+	return rand.Intn(delta) + min
```

Comment:
2 issues. first is that this vanity function is essentially equivalent to `rand.Intn(100)` and so you should just use that where you are calling this. second is that random number out of 100 is pretty likely to result in collisions and you've got no collision detection. why not use uuid generator?